frontend/assets/styles/ant-design-dark.less

```diff
@@ -1,101 +1,101 @@
 .dark {
-  @import '~ant-design-vue/lib/style/components';
-
-  @main: #222629;
-  @main-dark: #191c1e;
-  @main-light: #2e3338;
-  @text: #ddd;
-  @text-light: #fff;
-  @text-dark: #222;
-  @text-disabled: #999;
-
-  @primary: #1aab9b;
-  @info: #4ac8d2;
-  @success: #06d6a0;
-  @error: #ef476f;
-  @warn: #e55934;
-
-  @primary-color: @primary;
-  @processing-color: @info;
-  @info-color: @info;
-  @success-color: @success;
-  @error-color: @error;
-  @highlight-color: @error;
-  @warning-color: @warn;
-  @text-color: @text;
-  @text-color-dark: @text-dark;
-  @disabled-color: @text-disabled;
-  @text-color-secondary: @text-disabled;
-  @heading-color: @text-light;
-  @heading-color-dark: @text-dark;
-  @link-color: lighten(@primary, 10%);
-
-  @body-background: @main;
-  @component-background: @main-light;
-  @dropdown-selected-color: @text-color;
-  @item-active-bg: darken(@primary, 20%);
-  @item-hover-bg: darken(@primary, 20%);
-  @menu-highlight-color: @text;
-  @table-row-hover-bg: darken(@primary, 10%);
-
-  @descriptions-bg: @main-light;
-  @menu-bg: transparent;
-
-  @border-color-base: @main-dark;
-  @border-color-split: lighten(@main-light, 2%);
-
-  @background-color-light: @main-light;
-  @background-color-base: @main-light;
-
-  @checkbox-check-color: @main-light;
-
-  @layout-header-background: @main-dark;
-  @layout-body-background: @main;
-
-  // Alert
-  @alert-success-border-color: @success;
-  @alert-success-bg-color: darken(@success, 20%);
-  @alert-success-icon-color: @success;
-
-  @alert-info-border-color: @info;
-  @alert-info-bg-color: darken(@info, 30%);
-  @alert-info-icon-color: @info;
-
-  @alert-warning-border-color: @warn;
-  @alert-warning-bg-color: darken(@warn, 20%);
-  @alert-warning-icon-color: @warn;
-
-  @alert-error-border-color: @error;
-  @alert-error-bg-color: darken(@error, 20%);
-  @alert-error-icon-color: @error;
-
-  // Popover
-  @popover-bg: @component-background;
-
-  // fix certain styles in dark mode
-  .ant-alert.ant-alert-no-icon {
-    padding: 8px 15px;
-  }
-
-  .ant-popover-arrow {
-    border-color: @popover-bg !important;
-  }
-
-  .ant-modal-body {
-    padding: 0;
-  }
-
-  .ant-progress-circle-trail {
-    stroke: lighten(@main-light, 10%) !important;
-  }
-
-  .ant-radio-group {
-    display: flex;
-  }
-
-  .ant-alert-description {
-    display: block;
-  }
-
-  color-scheme: dark;
+    @import '~ant-design-vue/lib/style/components';
```

Comment:
I'm not sure this file linting was proper before (it had tab-width 2 before and have 4 now)